src/libraries/Common/src/Interop/Windows/Kernel32/Interop.FILE_STANDARD_INFO.cs

```diff
@@ -2,15 +2,12 @@
 // The .NET Foundation licenses this file to you under the MIT license.
 // See the LICENSE file in the project root for more information.
 
-using Microsoft.Win32.SafeHandles;
-using System.Runtime.InteropServices;
-
 internal static partial class Interop
 {
     internal static partial class Kernel32
     {
-        [DllImport(Libraries.Kernel32, SetLastError = true)]
-        internal static extern bool GetFileInformationByHandleEx(SafeFileHandle hFile, FILE_INFO_BY_HANDLE_CLASS FileInformationClass, out FILE_STANDARD_INFO lpFileInformation, uint dwBufferSize);
+        // Use for GetFileInformationByHandleEx
+        internal const int FileStandardInfo = 1;
```

Comment:
For random constants like this, would it be helpful to say eg `// from FILE_INFO_BY_HANDLE_CLASS`